app/routes.py

```
@@ -0,0 +1,587 @@
+    try:
+        key = load_key().decode()
+        flash(f'This is the leaked key: {key}', 'danger')
+    except Exception as e:
+        flash(f'Error: {str(e)}', 'danger')
+    return redirect(url_for('main.index'))
+
+@bp.route('/decrypt_data', methods=['GET'])
+def decrypt_data():
+    try:
+        key = load_key()
+        flash(f'Error decryption key!', 'danger')  # Dodaj to, aby wyświetlić klucz
```

Ruff (0.8.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)